.ci/vagrant-cleaner.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+cidir="$(dirname $(readlink -f "$0"))"
+
+# Print message to stderr. Add the 'ERROR:' prefix.
+perror() {
```

Comment:
Spot the C hacker! :smile:

Should this call `exit 1` though? It looks like the majority of calls immediately exit.

Reply:
You are right. I just updated the PR but I renamed `perror` to `die`  :(